presto-hive/src/main/java/com/facebook/presto/twitter/hive/MetastoreStaticClusterModule.java

```diff
@@ -11,8 +11,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.hive;
+package com.facebook.presto.twitter.hive;
```

Comment:
I think we should strive to only have things in the FB namespace that are contributed back. If it's useful to be in presto, we should contribute it back and remove the class from twitter.